l10n_it_fatturapa_in/wizard/wizard_import_fatturapa.py

```diff
@@ -322,11 +322,12 @@ def _prepareInvoiceLine(
                     ('child_ids', '=', False),
                 ], context=context)
             if not account_tax_ids:
-                raise orm.except_orm(
```

Comment:
no perche' in questo caso il controllo e' per il campo Natura quando l'iva e' a 0.0, quindi qui se non c'e' devono intervenire.